src/main/java/edu/kit/provideq/toolbox/meta/SubRoutineDefinition.java

```diff
@@ -4,9 +4,8 @@
  * Definition of a sub routine that holds basic information.
  *
  * @param type        type of the problem that is used in the sub routine
- * @param url         url of the problem to call the sub routine
  * @param description description of the sub routine call to provide information where and why it is
  *                    needed
  */
-public record SubRoutineDefinition(ProblemType type, String url, String description) {
+public record SubRoutineDefinition(ProblemType type, String description) {
```

Comment:
We still need the url as part of the record. See here for its usage: https://github.com/ProvideQ/toolbox-web/blob/develop/src/components/solvers/SolverPicker.tsx#L123

What we should do here instead is only require the problem type and description in the constructor for `SubRoutineDefinition` so the url part can be inferred from the type.